server/server.go

```diff
@@ -158,10 +167,13 @@ func NewDefaultDriver() *DefaultDriver {
 
 // ListenAndServe sets the address and handler on DefaultDriver's http.Server,
 // then calls ListenAndServe on it.
-func (dd *DefaultDriver) ListenAndServe(addr string, h http.Handler) error {
+func (dd *DefaultDriver) ListenAndServe(addr, certFile, keyFile string, h http.Handler) error {
```

Comment:
Consider making the drivers take in a [`*tls.Config`](https://pkg.go.dev/crypto/tls?tab=doc#Config) and then having the portable type provide a shortcut glue to use files. It is not uncommon to need to customize the `*tls.Config`, especially in scenarios with dynamically created (Let's Encrypt) certificates.

Reply:
This turned out not to be necessary; the `*tls.Config` can be given to drivers at construction time. For example, the `server.DefaultDriver` already has a `http.Server` in it that the user can access to set the `TLSConfig` field.